content/source/docs/registry/private.html.md

```diff
@@ -3,7 +3,7 @@ layout: "registry"
 page_title: "Terraform Registry - Private Registry"
 sidebar_current: "docs-registry-private"
 description: |-
-  Terraform can load private modules from private registries via Terraform Cloud.
+  Terraform Cloud allows Terraform to load private modules from private registries.
```

Comment:
I might leave this as it was. It's a super subtle distinction, but Terraform inherently has the ability to load modules from private registries, and Terraform Cloud provides one such registry. The edit makes it sound like Terraform Cloud is adding something to Terraform binary's capabilities, but I think it actually complies with Terraform binary's standards. Hope that makes sense :P

Reply:
it does! good catch - I'll change it back!